### PR TITLE
fix for default material metalness value

### PR DESCRIPTION
**Pull request by HexaField**
## References
closes #_insert number here_

### CODE REVIEW

File: packages/engine/src/gltf/GLTFState.tsx [hunk 1]
```diff
@@ -977,7 +978,16 @@ const CameraReactor = (props: { nodeIndex: number; documentID: string; entity: E
   return null
 }
 
-const defaultMaterial = () => new MeshStandardMaterial()
+const defaultMaterial = () =>
+  new MeshStandardMaterial({
+    color: 0xffffff,
+    emissive: 0x000000,
+    metalness: 1,
```
Comment (dinomut1) [inline packages/engine/src/gltf/GLTFState.tsx:985]: metalness should be 0 instead of 1 i think

**Review by dinomut1 — DISMISSED** [review]

File: packages/engine/src/gltf/GLTFState.tsx (hunk 1 above)
Comment (HexaField) [reply packages/engine/src/gltf/GLTFState.tsx:985]: GLTF specifies it should be 1 by default. same thing exists in the threejs GLTF loader. This is why certain things were loading with different colours than before.

Comment (HexaField) [reply packages/engine/src/gltf/GLTFState.tsx:985]: https://github.com/ir-engine/ir-engine/blob/fix-for-default-material-metalness-value/packages/engine/src/assets/loaders/gltf/GLTFLoaderFunctions.ts#L440

Comment (DanielBelmes) [reply packages/engine/src/gltf/GLTFState.tsx:985]: https://registry.khronos.org/glTF/specs/2.0/glTF-2.0.html#reference-material-pbrmetallicroughness I believe he is right yea

**Review by DanielBelmes — APPROVED** [review]
Approved on my front. Does seem to match default material of GLTF.

**Comment by dinomut1** [comment]
not resolved. metalness should never be 1 by default

**Pull request merged**